nevergrad/functions/multiobj/__init__.py

```diff
@@ -0,0 +1,140 @@
+from typing import Callable, List
```

Comment:
Please avoid any code in `__init__.py`, which by convention is rather used only for defining the imports from the package. You could use multiobj/core.py or functions.py or anything